config.json.example

```diff
@@ -94,8 +94,8 @@
     },
     "ITUNES_SHARED_SECRET": "aaaabbbbccccddddeeeeffff00001111",
     "EMAILS" : {
-        "COMMUNITY_MANAGER_EMAIL" : "community@example.com",
-        "TECH_ASSISTANCE_EMAIL" : "tech@example.com",
-        "PRESS_ENQUIRY_EMAIL" : "press@example.com"
+        "COMMUNITY_MANAGER_EMAIL" : "leslie@habitica.com",
+        "TECH_ASSISTANCE_EMAIL" : "admin@habitica.com",
+        "PRESS_ENQUIRY_EMAIL" : "leslie@habitica.com"
```

Comment:
Yes, in the screenshot you can't distinguish how it's been written in the code, but we have to do code reviews anyway for all PRs so we can check for hard-coded email addresses then. Viewing a screenshot is never a fully reliable way to ensure code has been written correctly.

It's more important that the screenshots show exactly what the text will look like in production. The screenshots will sometimes need to be approved by moderators or staff who don't have experience with reviewing the code, and they will want to see the real email address to do a full visual review and to be certain the text is correct. The screenshots will also sometimes also be used by Socialites and wiki editors for understanding new features and documenting them, including copying the screenshots to the wiki, and having fake email addresses in them will be confusing and could lead to misunderstandings.